src/haddock/modules/refinement/flexref/cns/flexref.cns

```diff
@@ -21,31 +21,29 @@ end if
 !==================================================================!
 
 evaluate ($saprotocol.iniseed=$iniseed)
-evaluate ($saprotocol.tadhigh_t=$tadhigh_t)
-evaluate ($saprotocol.t1_init=$tadinit1_t)
-evaluate ($saprotocol.t2_init=$tadinit2_t)
-evaluate ($saprotocol.t3_init=$tadinit3_t)
-evaluate ($saprotocol.t1_final=$tadfinal1_t)
-evaluate ($saprotocol.t2_final=$tadfinal2_t)
-evaluate ($saprotocol.t3_final=$tadfinal3_t)
-evaluate ($saprotocol.inter_rigid=$inter_rigid)
-evaluate ($saprotocol.inter_init_rigid=$init_rigid)
-evaluate ($saprotocol.inter_fin_rigid=$fin_rigid)
-evaluate ($saprotocol.inter_init_cool2=$init_cool2)
-evaluate ($saprotocol.inter_fin_cool2=$fin_cool2)
-evaluate ($saprotocol.inter_init_cool3=$init_cool3)
-evaluate ($saprotocol.inter_fin_cool3=$fin_cool3)
+evaluate ($saprotocol.tadhigh_t=$temp_high)
+evaluate ($saprotocol.t1_init=$temp_cool1_init)
+evaluate ($saprotocol.t2_init=$temp_cool2_init)
+evaluate ($saprotocol.t3_init=$temp_cool3_init)
+evaluate ($saprotocol.t1_final=$temp_cool1_final)
+evaluate ($saprotocol.t2_final=$temp_cool2_final)
+evaluate ($saprotocol.t3_final=$temp_cool3_final)
```

Comment:
The name of the parameters are more intuitive like this, we need to remember when writing the documentation that these are different from 2.x

Reply:
Will have to document all variables in the yaml file once we convert the cfg to yaml